Dockerfile

```diff
@@ -20,6 +20,10 @@ COPY . .
 # Uncomment the following line in case you want to disable telemetry during the build.
 # ENV NEXT_TELEMETRY_DISABLED 1
 
+ENV IMAGE_SERVICE_PORT=80
```

Comment:
I am not sure about this. Why don't we make .env file? This can be different in kube.